pysatNASA/instruments/timed_guvi.py

```diff
@@ -0,0 +1,356 @@
+# Instrument attributes
+
+platform = 'timed'
+name = 'guvi'
+tags = {'sdr-imaging': 'Level 1C imaging data',
+        'sdr-spectrograph': 'Level 1C spectrograph data'}
+inst_ids = {'high_res': list(tags.keys()),
+            'low_res': list(tags.keys())}
+
+pandas_format = False
+
+# ----------------------------------------------------------------------------
+# Instrument test attributes
+
+_test_dates = {
+    'high_res': {tag: dt.datetime(2005, 6, 28) for tag in tags.keys()},
+    'low_res': {tag: dt.datetime(2005, 6, 28) for tag in tags.keys()}}
+
+# ----------------------------------------------------------------------------
+# Instrument methods
+
+
+def init(self):
+    """Initialize the Instrument object with references and acknowledgements."""
+
+    rules_url = 'http://guvitimed.jhuapl.edu/home_guvi-datausage'
+    ackn_str = ' '.join(('Please see the Rules of the Road at', rules_url))
+    logger.info(ackn_str)
+    self.acknowledgements = ackn_str
+    self.references = ' '.join(('Paxton,L. J., Christensen, A. B., Humm,',
+                                'D. C., Ogorzalek, B. S., Pardoe, C. T.,',
+                                'Monison, D., Weiss, M. B., Cram, W.,',
+                                'Lew, P. H., Mabry, D. J., Goldstena,',
+                                'J. O., Gary, A., Persons, D. F., Harold,',
+                                'M. J., Alvarez, E. B., ErcoF, C. J.,',
+                                'Strickland, D. J., Meng, C.-I.',
+                                'Global ultraviolet imager (GUVI): Measuring',
+                                'composition and energy inputs for the NASA',
+                                'Thermosphere Ionosphere Mesosphere Energetics',
+                                'and Dynamics (TIMED) mission.',
+                                'Optical spectroscopic techniques and',
+                                'instrumentation for atmospheric and',
+                                'space research III. Vol. 3756.',
+                                'International Society for Optics',
+                                'and Photonics, 1999.'))
+
+    return
+
+
+def clean(self):
+    """Routine to return TIMED GUVI data cleaned to the specified level.
+
+    Note
+    ----
+    No cleaning currently available.
+
+    """
+    warnings.warn('no cleaning routines available for TIMED GUVI data')
+
+    return
+
+
+# ----------------------------------------------------------------------------
+# Instrument functions
+#
+# Use the default CDAWeb and pysat methods
+# Set the list_files routine
+
+fname = ''.join(('TIMED_GUVI_L1C{res:s}-disk-{mode:s}_{{year:04d}}{{day:03d}}',
+                 '{{hour:02d}}{{minute:02d}}{{second:02d}}',
+                 '-?????????????_REV??????_',
+                 'Av{{version:02d}}-??r{{revision:03d}}.nc'))
+
+res = {'low_res': '-2', 'high_res': ''}
+mode = {'sdr-imaging': 'IMG', 'sdr-spectrograph': 'SPECT'}
```

Comment:
Checking the website, I think this needs to be updated to "imaging" and "spectograph": https://cdaweb.gsfc.nasa.gov/pub/data/timed/guvi/levels_v13/level1c/.  How about:
```suggestion
mode = {tag: tag.split('-')[-1] for tag in tags.keys()}
```

Reply:
I just made a commit to fix this bug. Your suggestion was needed a few lines below. Thanks!